apps/editing-toolkit/editing-toolkit-plugin/readme.txt

```diff
@@ -40,6 +40,9 @@ This plugin is experimental, so we don't provide any support for it outside of w
 
 == Changelog ==
 
+= 2.8.13 =
+* Removes unnecessary site editor implemtation from the plugin. (https://github.com/Automattic/wp-calypso/pull/47301)
```

Comment:
Small typo with `implemtation`

Reply:
LOL oh no, now its going to trigger a rebuild on the diff.  🤣  all good though. good catch.

Reply:
oh! If it's a lot of trouble I don't think it's a big deal :P

Reply:
I think people understand what we're trying to say 😅